_maps_2024/29-aki.md

```diff
@@ -0,0 +1,11 @@
+---
+day: 29
+creator: Akira Di Sandro
+title: Overture Maps
+thumbnail: /assets/thumbnails/2024/29-Aki-Overture.png
+source: https://github.com/disandroa/30DayMappingChallenge2024/blob/main/scripts/Day29.R
+---
+
+View the [interactive map](assets/thumbnails/2024/29-Aki-Overture.html) of building classifications in Firenze, Italy.
+
+![Screenshot of map](assets/thumbnails/2024/29-Aki-Overture.png)
```

Comment:
:+1: 

Note that you could also use something like the [`iframe`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe) tag to embed the map directly on this page; something like:

```html
<iframe
  title="Building classifications in Firenze, Italy"
  src="assets/thumbnails/2024/29-Aki-Overture.html"></iframe>
```

Getting the sizing right on iframes can sometimes be tricky. I don't remember what the "i" in "iframe" stands for :shrug:. You can submit a new PR if you'd like to try it out.

Reply:
I think I will submit as is right now. will try to clean this up when I have more time🥲